.infrastructure/55_cognito.tf

```diff
@@ -11,7 +11,8 @@ module "cognito_custom_message_function" {
   create_current_version_allowed_triggers = false
 
   environment_variables = {
-    DOMAIN = var.dns_domain_name
+    DOMAIN             = var.dns_domain_name
```

Comment:
Add a TODO to explain why we have to add all the environments

Reply:
I made some changes